Source/SwiftLintFramework/Protocols/Rule.swift

```diff
@@ -32,13 +43,40 @@ extension ParameterizedRule {
     }
 }
 
+extension ParameterizedRule where Self: ConfigurableRule, ParameterType == Int {
```

Comment:
I think this can be generalized to avoid the constraint on `Int`. What happens when you replace the references to `Int` in this extension to `ParameterType`?